includes/functions/functions_prices.php

```diff
@@ -490,8 +494,6 @@ function zen_get_discount_calc($product_id, $attributes_id = false, $attributes_
 
     $new_products_price = zen_get_products_base_price($product_id);
     $new_special_price = zen_get_products_special_price($product_id, true);
-    $new_sale_price = zen_get_products_special_price($product_id, false);
```

Comment:
Yes, in the spirit of reducing queries, this would be one to be removed from the code as it currently stands; however, as identified in #1404, there are some discrepancies that would need this value to be "returned" for use unless an alternative solution has been developed.

Reply:
Sigh. Why do we have 4 PRs all about the same subject, all going in different directions?

Reply:
Okay, I'm gonna back out this change.